requirements.txt

```diff
@@ -34,7 +34,7 @@ subliminal==2.1.0
 tmdbsimple==2.8.0
 tornado==6.1
 tornroutes==0.5.1
-trakt==3.1.0
+trakt==3.4.0
```

Comment:
**trakt** is not maintained [anymore](https://github.com/moogar0880/PyTrakt/issues/199) and crashes (#10762).
You should use the updated fork [pytrakt](https://pypi.org/project/pytrakt/) 3.4.20

Reply:
Thanks for the heads-up! Would be great if you could help testing this branch: https://github.com/pymedusa/Medusa/pull/11173